.github/workflows/main.yml

```diff
@@ -16,6 +16,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
```

Comment:
I'm confused by this change. This is the default when building a ref but for other actions it will be set appropriately. So why the override?

Reply:
Removed this on both jobs.